### ISSUE TITLE
SSLerror self signed certificate

### ISSUE THREAD

**Issue by riddertommie** [jerrymakesjelly/autoremove-torrents#94]
I have a self-singed certificate and i think this is the reason that it does not work.

config:

my_task:
  client: qBittorrent
  host: https://127.0.0.1/
  username: admin
  password: XXXxxxx
  strategies:
    my_strategy:
      categories: tv-sonarr
      remove: seeding_time > 1209600 or ratio > 1
  delete_data: true

error:

sudo autoremove-torrents
Thu, 29 Oct 2020 12:23:24 autoremovetorrents.main INFO: Auto Remove Torrents 1.5.2
Thu, 29 Oct 2020 12:23:24 autoremovetorrents.main INFO: Loading configurations...
Thu, 29 Oct 2020 12:23:24 autoremovetorrents.main INFO: Found 1 task(s) in the file.
Thu, 29 Oct 2020 12:23:24 autoremovetorrents.task INFO: Running task 'my_task'...
Thu, 29 Oct 2020 12:23:24 autoremovetorrents.main ERROR: requests.exceptions.SSLError: HTTPSConnectionPool(host='127.0.0.1', port=443): Max retries exceeded with url: //api/v2/app/webapiVersion (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])")))
Thu, 29 Oct 2020 12:23:24 autoremovetorrents.main ERROR: Task my_task fails.

Or am I doing something wrong?

**Comment by jerrymakesjelly** [comment]
Yes, as you think, this error is due to the self-signed certificate.

I will add the self-signed certificate support in the next version. Thanks for your feedback.

**Comment by riddertommie** [comment]
Thanks for letting me/us know!
